core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -83,6 +86,7 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
   inClassroomPage: boolean = false;
   showLanguageSelector: boolean = false;
   standardNavIsShown: boolean = false;
+  featureEnable: boolean;
```

Comment:
It's a bit unclear what this variable represents. Could you renaming it such that it is self-explanatory?

Reply:
Removed as unused